src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.controller;
+
+import java.util.NoSuchElementException;
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserResponseMapper userDtoResponseMapper;
```

Comment:
Use abstraction (DtoRequestMapper and DtoResponseMapper) and in constructors as well